src/config/environment.json

```diff
@@ -0,0 +1,24 @@
+{
+    "type": "firstDiscovery.server.environment.config",
+    "options": {
+        "gradeNames": ["gpii.firstDiscovery.server.configurator"],
+        "port": "{environment}.env.FIRST_DISCOVERY_SERVER_TCP_PORT",
+        "preferencesConfig": {
+            "securityServer": {
+                "port": "{environment}.env.GPII_OAUTH2_TCP_PORT",
```

Comment:
Based on the channel discussion the IoC expression will be {gpii.resolvers.env}.vars.someName.
Note that the property containing the variables changes from "env" to "vars".

https://botbot.me/freenode/fluid-work/2015-12-14/?msg=56128841&page=1